test/library-unit-tests.ts

```diff
@@ -1,8 +1,10 @@
 import * as elementData from '../src/utils/elementData';
 import * as mockData from './resources/webdriver-mock-data';
 import { Buttons, Library } from '../src/modules/library';
-import assert = require('assert');
 import nock = require('nock');
+import chai = require('chai');
```

Comment:
should these be imports instead of require?

Reply:
if `chai` exports a named function called `expect` you can import it like this: 

```
import { expect } from 'chai';
```

Reply:
Fixed in 59afaf3